cypress-tests/cypress/e2e/PaymentTest/00020-Variations.cy.js

```diff
@@ -331,7 +331,8 @@ describe("Corner cases", () => {
     });
 
     it("Capture call", () => {
-      let data = getConnectorDetails(globalState.get("commons"))["card_pm"][
+      let connectorId = globalState.get("connectorId");
+      let data = getConnectorDetails(connectorId === "worldpay" ? connectorId : globalState.get("commons"))["card_pm"][
```

Comment:
is there any other way to achieve the same?
asking so because we do not want to hardcode `tests` to a specific connector and prefer to have tests as generic as possible. a specific should not be an exception.

same applies to the changes done in file.

also, i highly doubt this will not work. it should be `connectorId["card_pm"]["CaptureCapturedAmount"]` instead of just `connectorId`

Reply:
Connector specific data should always be fetched from respective connectors, and the fallback logic can handle fetching the data from `Commons.js` wherever it's needed

Reply:
on variations, most of the tests never hit a connector until now and hence we directly used `commons`. now that i think of, even if you remove the hardcoded `commons` and replace that with `connectorId`, it should work. the reason being, the `Utils.js` in `PaymentUtils` merges the `<connector_name>.js` with `commons.js` before the test is run.

Reply:
Yes!